packages/redux-toolbelt/README.md

```diff
@@ -65,32 +70,34 @@ https://codesandbox.io/s/github/welldone-software/redux-toolbelt-demo
 
 ## API Reference
 
-
-
 ### `makeActionCreator()`
-Create an FSA complient action creator that exposes its `TYPE` as static member.  
-This can help force type-safty without adding alot of verbose code and constants.  
+
+Creates an [FSA](https://github.com/redux-utilities/flux-standard-action) compliant action creator that exposes its `TYPE` as a static member.  
+This can help force type-safety without adding a lot of verbose code and constants.
 
 All produced actions will have a `type`, `payload` and `meta` properties.
+
 ```js
-const myAction = makeActionCreator('MY_ACTION')
+const myAction = makeActionCreator('MY_ACTION');
```

Comment:
Please notice that the style encouraged by welldone-software is not to use `;`, as they are following `airbnb`'s style guide.
Likewise for spaces inside objects: `{item}` over `{ item }`, etc.